redaxo/src/core/lib/form/form_base.php

```diff
@@ -998,6 +1002,33 @@ public function getMessage()
         return $message;
     }
 
+    /**
+     * @return void
+     */
+    public function setFormAttribute(string $attributeName, ?string $attributeValue)
+    {
+        $attributeName = preg_replace('/[^\w\d\-]/', '', strtolower($attributeName));
+
+        if (0 === mb_strlen($attributeName)) {
+            return;
+        }
+
+        if (null === $attributeValue && array_key_exists($attributeName, $this->formAttributes)) {
```

Comment:
```suggestion
        if (null === $attributeValue) {
```

Bei `null` sollte es immer hier ins `if` und dann abbrechen. Ansonsten würde am Ende ein leerer Wert gesetzt werden, wenn es das Attribut vorher nicht gab.